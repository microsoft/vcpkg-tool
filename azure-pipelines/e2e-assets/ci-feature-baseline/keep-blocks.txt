# first line
# second line

b = fail
c = fail
a = fail


# new block

b = cascade
c = cascade
a = cascade