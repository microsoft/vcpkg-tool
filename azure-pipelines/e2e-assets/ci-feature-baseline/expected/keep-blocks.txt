# first line
# second line

a = fail
b = fail
c = fail


# new block

a = cascade
b = cascade
c = cascade
